packages/flame_spine/example/assets/spineboy/LICENSE

```diff
@@ -0,0 +1,8 @@
+Copyright (c) 2013, Esoteric Software
```

Comment:
should we use our standard, MIT license?

Reply:
This is the license for the spineboy asset, it is not ours.